### PR TITLE
Update setupcfg dependencies

### PR DESCRIPTION
**Pull request by ColeDCrawford**
Even after installing wheel I was getting this error: https://stackoverflow.com/questions/34819221/why-is-python-setup-py-saying-invalid-command-bdist-wheel-on-travis-ci

### CODE REVIEW

**Comment by ColeDCrawford** [comment]
Added `python-magic` and `Deprecated` as requirements as well. Previously, if you didn't have them otherwise installed in your project, they won't be installed and using `IIIFingest` would fail.

File: setup.cfg [hunk 1]
```diff
@@ -28,6 +28,10 @@ install_requires =
     requests
     shortuuid
     pyIIIFpres
+    python-magic
+    Deprecated
```
Comment (arthurian) [inline setup.cfg:32]: Good call on fixing these missing requirements.

**Review by arthurian — APPROVED** [review]
@ColeDCrawford The changes look fine to me. I just had one question about whether `setup_requires` should be replaced in favor of the `build-system` table. :+1:

File: setup.cfg [hunk 2]
```diff
@@ -28,6 +28,10 @@ install_requires =
...
+setup_requires =
```
[the remark this answers is not in the document]
Comment (ColeDCrawford) [reply setup.cfg:33]: Good call.

Right now we're building with `python setup.py sdist bdist_wheel`. It looks like we would need to add `PyPA`'s `build` and then we could do `python3 -m build`. That seems to work for me.

[the remark this answers is not in the document]
Comment (jaguillette) [reply setup.cfg:33]: Hm, interesting. This article was a bit helpful in understanding PEP 518: https://chadsmith-software.medium.com/pep-517-and-518-in-plain-english-47208ca8b7a6. One thing I was looking for was whether version numbers can be included in the `requires` section, and it seems like normal version specification syntax works there.

[the remark this answers is not in the document]
Comment (ColeDCrawford) [reply setup.cfg]: With `pyproject.toml` and `setup.cfg` in place, do we still need `setup.py`?

I was able to build this without it:
```
➜  lts-iiif-ingest-service git:(fix-setupcfg) rm -rf iiif-lts-env 
➜  lts-iiif-ingest-service git:(fix-setupcfg) python3 -m venv iiif-lts-env
➜  lts-iiif-ingest-service git:(fix-setupcfg) rm setup.py 
➜  lts-iiif-ingest-service git:(fix-setupcfg) ✗ source iiif-lts-env/bin/activate
(iiif-lts-env) ➜  lts-iiif-ingest-service git:(fix-setupcfg) ✗ pre-commit install                 
pre-commit installed at .git/hooks/pre-commit
(iiif-lts-env) ➜  lts-iiif-ingest-service git:(fix-setupcfg) ✗ python3 -m build                   
* Creating virtualenv isolated environment...
* Installing packages in isolated environment... (setuptools>=42, wheel)
* Getting build dependencies for sdist...
* Building sdist...
* Building wheel from sdist
* Creating virtualenv isolated environment...
* Installing packages in isolated environment... (setuptools>=42, wheel)
* Getting build dependencies for wheel...
* Installing packages in isolated environment... (wheel)
* Building wheel...
Successfully built IIIFingest-1.1.1.tar.gz and IIIFingest-1.1.1-py3-none-any.whl
```

**Review by arthurian — APPROVED** [review]
The changes looks good @ColeDCrawford! 👍

**Pull request merged**